[Request] swift-language-git

- Link to the package(s) in AUR: https://aur.archlinux.org/packages/swift-language-git

- Utility this package has for you:

  This package contains the swift programming language.

- Do you consider this package(s) to be useful for **every** chaotic user?:

  - [ ] YES
  - [ ] No, but yes for a great amount.
  - [X] No, but yes for a few.
  - [ ] No, it's useful only for me.

- Do you consider this package(s) to be useful for feature testing/preview (e.g: mesa-aco, wine-wayland)?:

  - [ ] YES
  - [X] NO

- Are you sure we don't have this package already (test with `pacman -Ss <pkgname>`)?:

  - [X] YES

- Have you tested if this package builds in a clean chroot?:

  - [ ] YES
  - [X] NO

- Does the package's license allows us to redistribute it?:

  - [X] YES
  - [ ] No clue.
  - [ ] No, but the author doesn't really care, it's just for bureaucracy.

- Have you searched the [issues](https://github.com/chaotic-aur/packages/issues) to ensure this request is new (not duplicated)?:

  - [X] YES

- Have you read the [README](https://github.com/chaotic-aur/packages#banished-and-rejected-packages) to ensure this package is not banned?:

  - [X] YES

`-- Install configuration: "Release" CMake Error at cmake_install.cmake:46 (file): file cannot create directory: /home/main-builder/pkgwork/pkg/swift-language-git/usr/lib/swift-git/bin. Maybe need administrative privileges. Call Stack (most recent call first): /home/main-builder/pkgwork/src/build/buildbot_linux/llvm-linux-x86_64/tools/cmake_install.cmake:66 (include) /home/main-builder/pkgwork/src/build/buildbot_linux/llvm-linux-x86_64/cmake_install.cmake:77 (include) `

Does not seem like a toolbox issue to me 🤔

> `-- Install configuration: "Release" CMake Error at cmake_install.cmake:46 (file): file cannot create directory: /home/main-builder/pkgwork/pkg/swift-language-git/usr/lib/swift-git/bin. Maybe need administrative privileges. Call Stack (most recent call first): /home/main-builder/pkgwork/src/build/buildbot_linux/llvm-linux-x86_64/tools/cmake_install.cmake:66 (include) /home/main-builder/pkgwork/src/build/buildbot_linux/llvm-linux-x86_64/cmake_install.cmake:77 (include) `
> 
> Does not seem like a toolbox issue to me thinking

The package is trying to write the files to `"$pkgdir"` during `build()`...

--- a/ufscar-hpc/daily.1.txt
+++ b/ufscar-hpc/daily.1.txt
@@ -52,6 +52,9 @@ freecad-git
 # Issue 793
 dogecoin-qt
 
+# Issue 799
+swift-language-git
+
 # Issue 808
 codelite-unstable
 
@@ -87,20 +90,17 @@ lyx
 # Issue 1404
 swiftshader-git
 
-# Issue 779
-swift-language-git:https://github.com/chaotic-aur/pkgbuild-swift-language-git
-
 # Issue 1632
 kodi-git:https://aur.archlinux.org/kodi-git.git
-
-# Issue 2175
-kodi-nexus-git
 
 # Issue 1652
 linux-lqx
 
 # Issue 1784
 luckybackup
+
+# Issue 2175
+kodi-nexus-git
 
 # Issue 2278
 lapce-git

